pom.xml

```diff
@@ -15,7 +15,7 @@
   </description>
   <artifactId>naginator</artifactId>
   <packaging>hpi</packaging>
-  <version>1.17.3-SNAPSHOT</version>
+  <version>1.18.0-SNAPSHOT</version>
```

Comment:
I decided to change the minor version even though this doesn't add new features to naginator as the next release requires newer version of Jenkins (1.554 -> 1.565).